public/translations/pt-BR.json

```diff
@@ -88,7 +88,14 @@
     "unknown": "Opa. Algo de errado aconteceu =(.",
     "accountNotFound": "Conta não encontrada. Tem certeza que criou uma conta com essa chave?",
     "phone": "Por favor insira um número de telefone válido",
-    "zipcode": "CEP inválido"
+    "zipcode": "CEP inválido",
+    "contracts": {
+      "verifyclaim": {
+        "Can't vote on already verified claim": "Não é possível votar em uma reivindicação já verificada",
+        "This is action is already completed, can't verify claim": "Esta ação já foi concluída, não é possível verificar a reivindicação",
+        "There are no usages left for this action": "Não há usos restantes para esta ação"
```

Comment:
we can also add a `error.unknown` for unhandled error cases, with the same general message we have on `master`

Reply:
Yep, I already use `error.unknown` as a [fallback](https://github.com/cambiatus/frontend/blob/32f81c6d47a2636b2d64d637b389761f4ab57937/src/elm/Eos/EosError.elm#L31) while parsing JSON with errors.

Reply:
Yeah, just talking about that we have no translations yet (I know its a draft)! nice one!